index.html

```diff
@@ -1430,9 +1441,48 @@
 
 ## Harassment
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Online <dfn>harassment</dfn> is the "pervasive or severe targeting of an individual or group online
+through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the Web,
+particularly via social media. Harassment may be more severe and its consequences more impactful for
+LGBTQ people and women [[?PEW-Harassment]].
+
+<a>Harassment</a> is both a violation of privacy itself and can be a consequence of other violations
+of privacy. 
+
+Abusive online behavior may include: sending <a href="#unwanted-information">unwanted
+information</a>; directing others to contact or bother a person ("dogpiling"); disclosing sensitive
+information about a person; posting false information about a person; impersonating a person;
+insults; threats; and hateful or demeaning speech.
+
+Disclosure of contact information can be used, including by a separate attacker, to send often
+persistent unwanted information that amounts to harassment. Disclosure of location information can
+be used, including by a separate attacker, to intrude on a person's physical safety or space.
```

Comment:
Might it be worth mentioning doxing and swatting?

Reply:
added an explicit doxxing reference
swatting is included in the PEN reference, and would certainly be an example of physical intrusion, but I don't know if we need an explicit reference to that as well